otter/grade/containers.py

```diff
@@ -218,12 +223,18 @@ def kill_container():
             scores = dill.load(f)
 
         scores_dict = scores.to_dict()
-        scores_dict["percent_correct"] = scores.total / scores.possible
+        
+        scores_dict["percent_correct"] = round(scores.total / scores.possible,4)
+        scores_dict["total_points_earned"] = scores.total
 
+        pts_poss_dict = {t: [scores_dict[t]["possible"]] if type(scores_dict[t]) == dict else scores_dict[t] for t in scores_dict}
```

Comment:
The else clause in this dict comprehension is unnecessary. You can change this to

```
{t: [scores_dict[t]["possible"]] for t in scores_dict}
```

since each value should always be a dict.

And can you also do the same for `scores_dict` on the next line?

Reply:
I moved the percent_correct and points_possible attributes to after the dictionary creation to avoid having to check to see if the scores_dict[t] was a dictionary. I added this in on the last PR which makes no sense. I am pulling the scores and points possible first from the scores_dict and then adding in the totals now.